wdl-lint/src/v1/import_placement.rs

```diff
@@ -0,0 +1,118 @@
+//! A lint rule for import placements.
+
+use wdl_ast::v1::ImportStatement;
+use wdl_ast::v1::StructDefinition;
+use wdl_ast::v1::TaskDefinition;
+use wdl_ast::v1::Visitor;
+use wdl_ast::v1::WorkflowDefinition;
+use wdl_ast::AstNode;
+use wdl_ast::Diagnostic;
+use wdl_ast::Diagnostics;
+use wdl_ast::Span;
+use wdl_ast::ToSpan;
+use wdl_ast::VisitReason;
+
+use super::Rule;
+use crate::Tag;
+use crate::TagSet;
+
+/// The identifier for the import placement rule.
+const ID: &str = "ImportPlacement";
+
+/// Creates a "misplaced import" diagnostic.
+fn misplaced_import(span: Span) -> Diagnostic {
+    Diagnostic::warning("misplaced import")
+        .with_rule(ID)
+        .with_highlight(span)
+        .with_fix("move this import so that it comes directly after the version statement")
```

Comment:
Does `directly after` contradict the requirement for a blank line? When I read this "fix" message, I think the solution would be:
```
version 1.0
import "foo"
```

Reply:
Agreed, I'll remove the "directly" from the message.

Reply:
How about:

```suggestion
        .with_fix("move this import so that it comes after the version statement but before any document items")
```

Reply:
LGTM